embed/embed.go

```diff
@@ -171,11 +171,20 @@ func Run(conf *Config) {
 			// fs root.
 			local = local[1:]
 		}
+
+		var children = "[]string{"
+		childrenFileInfos, _ := ioutil.ReadDir(local)
+		for _, filename := range childrenFileInfos {
+			children += "\"" + filename.Name() + "\", "
```

Comment:
use fmt.Sprintf with `%q` to correctly escape. also this will need some newlines and tabs between each child (that is, put each one on its own line and make sure the output will be fmt'd correctly.